geoh5py/ui_json/input_file.py

```diff
@@ -303,6 +304,7 @@ def validation_options(self):
             return {
                 "update_enabled": True,
                 "ignore_list": (),
+                "disable_promotion": False,
```

Comment:
Since we are going to turn off validations on input file pretty soon, can we somehow make this an attribute of the class?

Reply:
done